Features.md

```diff
@@ -13,24 +13,46 @@
    * Profile Photo
    * Email (for authentication related)
    * Hobbies
-   * NUmber (for authentication related)
+   * Number (for authentication related)
    * Kind of places love to visit
    * Current location
-   * User Travel Histroy (the places the user has been to)
```

Comment:
We have to maintain the travel history of the user so User Travel History should be there.